dev-tools/e2e/certs.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+set -eu
+
+REPO_ROOT=$(cd $(dirname $(readlink -f "$0"))/../.. && pwd)
+CERT_DIR=${REPO_ROOT}/build/e2e-certs
+
+mkdir -p ${CERT_DIR}
+
+# Create CA
+openssl req -x509 \
+    -sha256 -days 356 \
+    -nodes \
+    -newkey rsa:2048 \
+    -subj "/CN=e2e-test-ca" \
+    -keyout ${CERT_DIR}/e2e-test-ca.key -out ${CERT_DIR}/e2e-test-ca.crt
+
+# Make encrypted private key
+echo -n abcd1234 > ${CERT_DIR}/passphrase
+
+openssl genpkey -algorithm RSA \
```

Comment:
I'm having issues with openssl, fleet-server is unable to use the cert/key pair i make here (yet it works with `elasticsearch-certutil`).

i'll look into this more later